ci/jenkins/test.sh

```diff
@@ -608,7 +608,6 @@ function deliver_antrea {
             kind load docker-image antrea/antrea-agent-ubuntu:$BUILD_TAG --name ${KIND_CLUSTER}
             kind load docker-image antrea/antrea-controller-ubuntu:$BUILD_TAG --name ${KIND_CLUSTER}
             kind load docker-image antrea/flow-aggregator:latest --name ${KIND_CLUSTER} 
-            kubectl config use-context kind-${KIND_CLUSTER}
```

Comment:
I understand that this was not correct, but how do we select the correct context? Do we use different Kubeconfig files and we set the `KUBECONFIG` env variable?
I guess we are assuming that for Kind tests, we will not run any kubectl command after this?

Reply:
The line is still valid for kind-flexible-ipam tests  ` kubectl config use-context kind-${KIND_CLUSTER}`
Need to put a check as currently kubeconfig will only be used for running multiple kind jobs on kind test bed and not kind-flexible-ipam tests.

Reply:
We were already exporting KUBECONFIG env variable to the KUBECONFIG_PATH in the test.sh script, so it was unnecessary to set context son because setting KUBECONFIG env variable would satisfy our purpose.